lib/msal-node/docs/v5-migration.md

```diff
@@ -0,0 +1,33 @@
+# Migrating from MSAL Node v4 to v5
```

Comment:
[This is the existing migration guide](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/msal-v5/lib/msal-node/docs/v4-migration.md) (it needs to be updated to reflect that we're jumping from v3 directly to v5, there is no and there will be no v4) can you put this there and/or make the necessary updates?

Reply:
done